Gems/Atom/Tools/ShaderManagementConsole/Code/Source/ShaderManagementConsoleApplication.cpp

```diff
@@ -47,27 +47,29 @@ void InitShaderManagementConsoleResources()
 
 namespace ShaderManagementConsole
 {
+    static const char* GetBuildTargetName()
+    {
+#if !defined(LY_CMAKE_TARGET)
+#error "LY_CMAKE_TARGET must be defined in order to add this source file to a CMake executable target"
+#endif
+        return LY_CMAKE_TARGET;
+    }
+
```

Comment:
[nit] This function appeared a couple of times. I wonder if it's worth moving it to a common place.

Reply:
This function appears once per application/project. It pulls the name of the executable target from a define in CMAKE and would not be available at a lower level. We could get rid of the function completely and just use LY_CMAKE_TARGET